### ISSUE TITLE
Improperly emulating links

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/Adapt-accessibilityfixes#97]
Issue: The "Skip to transcript" button should be a link (<a> or role="link") since it's an anchor, and it should be clearly styled like a link. Its currently not discernible in the text.

**Comment by GGD4V** [comment]
.aria-label.js-skip-to-transcript should be transformed to an anchor or role link and have proper formating.

**Comment by MeD-Owner** [comment]
Could it be a button? Can we make it look nice?

**Comment by GGD4V** [comment]
fixed https://github.com/MeD-DMC/adapt-contrib-esdc-college-theme/commit/04c58491b4c2a0e445caca2562d899dcc49f50ed
